tetris.py

```diff
@@ -285,7 +228,6 @@ def current_block_mover(self):
                 )
             )
             self.full_line_clear()
-            self.new_block()
         else:
             x, y = self.current_block_center
             self.current_block_center = (x, y + 1)
```

Comment:
And return `False` when a new block should not be added:

```suggestion
            self.current_block_center = (x, y + 1)
            return False  # no new block
```